C/deserialize.c

```diff
@@ -24,17 +24,17 @@ static int32_t getWord32Array(uint32_t* result, const size_t len, bitstream* str
     if (bits16 < 0) return bits16;
```

Comment:
In 9d3240e4468533acd17e2d14fc1eae85edf42ef4:

Should cast `bits16` here (and three lines above).

Reply:
If only we had compiler warnings ...

Reply:
Fixed.